terraform/18f.gov.tf

```diff
@@ -249,7 +307,25 @@ resource "aws_route53_record" "18f_gov_continua11y_18f_gov_cname" {
   records = ["dwz1sfldhro5q.cloudfront.net"]
 }
 
-# Configured with CDN Broker
+resource "aws_route53_record" "18f_gov_digitalaccelerator_18f_gov_a" {
+  zone_id = "${aws_route53_zone.18f_gov_zone.zone_id}"
+  name = "digitalaccelerator.18f.gov."
+  type = "A"
+  alias {
+    name = "dyumdy5yvu23d.cloudfront.net."
+    zone_id = "Z2FDTNDATAQYW2"
+    evaluate_target_health = false
+  }
+}
+
+resource "aws_route53_record" "18f_gov_dnssec_18f_gov_ns" {
+  zone_id = "${aws_route53_zone.18f_gov_zone.zone_id}"
+  name = "dnssec.18f.gov."
+  type = "NS"
+  ttl = 300
```

Comment:
That's good for quicker uptake. Once the migrations are all complete, let's come back and revise these to be longer, but that's outside the scope of this pull request. (cc @jcscottiii/@datn)